### PR TITLE
Update HOWTO-bn.md (bengali review requested)

### PR DESCRIPTION
**Pull request by nababuddin**
What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

For resources
Description
This PR adds a "How to at a Glance" document in Bengali language for the Free-Programming-Books repository. The document provides an overview of how to contribute to the repository and serves as a quick guide for both new and experienced contributors.

Why is this valuable (or not)?
This addition is valuable because it makes the repository more inclusive and accessible to Bengali-speaking contributors. It lowers the entry barrier for new contributors by providing essential resources and guidance, and it enhances the overall user experience.

How do we know it's really free?
The content is translated and adapted from the original English version, and no restrictions or paid resources are mentioned. It is consistent with the repository's goal of providing free programming resources.

For book lists, is it a book? For course lists, is it a course? etc.
This is not a book or course; it's a document that serves as a guide for contributors.

Checklist:
 Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
 [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
 Include author(s) and platform where appropriate.
 Put lists in alphabetical order, correct spacing.
 Add needed indications (PDF, access notes, under construction).
 Used an informative name for this pull request.
Follow-up
Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
asking for bengali review

**Review by kazisean — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books! and for review!

File: docs/HOWTO-bn.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+* [গিটহাব হ্যালো ওয়ার্ল্ড](https://docs.github.com/en/get-started/quickstart/hello-world) *(in english)*
+* [ইউটিউব - গিটহাব টিউটোরিয়াল প্রাথমিকদের জন্য](https://www.youtube.com/watch?v=0fKg7e37bQE) *(in english)*
+* [ইউটিউব - গিটহাব রিপোজিটরি ফর্ক করা এবং পুল রিকোয়েস্ট সাবমিট করা](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(in english)*
+* [ইউটিউব - মার্কডাউন ক্র্যাশ কোর্স](https://www.youtube.com/watch?v=HUBNt18RFbo) *(in english)*
 
 
-কোন প্রশ্ন করতে দ্বিধাবোধ করবেন না। সব কন্ট্রিবিউটরই প্রথম PR থেকে শুরু করেছিল। তাহলে কেন না আমাদের [large, growing](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) community তে যোগদান করুন!
+কোন প্রশ্ন করতে দ্বিধাবোধ করবেন না; সমস্ত কন্ট্রিবিউটর তাদের প্রথম PR দিয়ে শুরু করেছিলেন। তাহলে, কেন আমাদের বড় এবং ক্রমবর্ধমান কমিউনিটি তে যোগদান করছেন না? [community](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) তে যোগদান করুন!
```
Comment (huzaifaalmesbah) [inline docs/HOWTO-bn.md:21]: সমস্ত থেকে সকল দিলে বেশি ভালো হত

Comment (nababuddin) [reply docs/HOWTO-bn.md:21]: I think its fine

**Pull request merged**